programs/sll_sum_lists.py

```diff
@@ -0,0 +1,57 @@
+# Sum lists (page 95. 2.5)
+# input 2 sll exem. (2 -> 1 -> 6)  (2 -> 5 -> 3)   its equal to 612+352 = 964
+# return the sum as a sll ( 4 -> 6 -> 9)
+
+from single_linked_lists import Node
+
+
+def sum_nodes(a: Node, b: Node):
+    print(a)
+    print(b)
+
+    head = Node(0)
+    resul = head
+    ten = 0
+
+    while a or b or ten:
+        if a:
+            valuea = a.data
```

Comment:
Rename for legibility. Same elsewhere
```suggestion
            value_a = a.data
```